TUnit.Templates.Tests/Snapshots/InstantiationTest.TUnit.AspireStarter._.verified/TUnit.AspireStarter/TUnit.AspireStarter.TestProject/GlobalSetup.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Here you could define global logic that would affect all tests
+
+// You can use attributes at the assembly level to apply to all tests in the assembly
+
+using Aspire.Hosting;
+
+[assembly: Retry(3)]
+[assembly: System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage]
+
+namespace TUnit.AspireStarter.TestProject;
+
+public class GlobalHooks
+{
+    public static DistributedApplication? App{ get; private set; }
```

Comment:
If we know we're setting it in a before method we can avoid the nullability on the type. Also spacing issue with the brace.
```suggestion
    public static DistributedApplication App { get; private set; } = null!;
```